README.md

```diff
@@ -42,12 +42,12 @@ Need help?
 
 If you are stuck somewhere and can't figure out how to fix the problem(s) so 
 that you can run Tatoeba properly on your machine, don't hesitate to post a 
-message in our [Google group][5], ask in our [Gitter chatroom][6], or in our 
-IRC channel (server: freenode, channel: #tatoeba).
+message in our [Google group][5], ask in our [XMPP chatroom][6],
+[tatoeba@chat.tatoeba.org](xmpp:tatoeba@chat.tatoeba.org?join).
 
 [1]: https://github.com/Tatoeba/tatoeba2/wiki/Joining-the-dev-team
 [2]: https://github.com/Tatoeba/imouto
 [3]: https://github.com/Tatoeba/imouto/blob/master/README.md#imouto-for-development
 [4]: https://github.com/Tatoeba/tatoeba2/wiki/How-to-install-Tatoeba
 [5]: http://groups.google.com/group/tatoebaproject
-[6]: https://gitter.im/Tatoeba/tatoeba2?utm_source=share-link&utm_medium=link&utm_campaign=share-link
+[6]: https://chat.tatoeba.org/
```

Comment:
I just noticed you also edited the README. I wouldn't remove the link to Gitter just yet but simply replace the IRC part by XMPP.

In other words:
1. Keep the Gitter badge at the top. Feel free to add another badge for XMPP though.
2. Change the **Need help?** text as followed: "... don't hesitate to post a message in our [Google group][1], ask in our [Gitter chatroom][2], or in our [XMPP chatroom][3]."

[1]: http://groups.google.com/group/tatoebaproject
[2]: https://gitter.im/Tatoeba/tatoeba2?utm_source=share-link&utm_medium=link&utm_campaign=share-link
[3]: https://chat.tatoeba.org/